### DISCUSSION TITLE
Feature request: Import into a CalDAV calendary synced through DAVx5

### DISCUSSION THREAD

**Discussion by sihma** [bitfireAT/icsx5#49] [Ideas]
I have a use case which I though could be handled with ICSx5 yet actually can't.

TL;DR: Could ICSx5 import .ics file into an existing CalDAV calendar instead of importing as a read-only calendar.

I use a meeting management platform to allow people to book meetings with me (cal.com). This platforms checks my CalDAV calendar to show my availability to people.

I have another platform where people book meetings with me. This platform provide an .ics feed that I automatically refresh using ICSx5.

So: my ideal use case, would be that I can import the .ics feed into my CalDAV calendar automatically, so cal.com can see I'm not available at these times. I don't know if I should ask this here or on the DAVx5 forum. It seems that this function would be complicated to add.

Thanks for your feedback. I also asked the cal.com team if it would be possible for them to add support for ICS feeds.

**Comment by rfc2822** [comment]
Hi,

I wouldn't put this feature into the client. It's better suited in the CalDAV server and often called "subscription" there.

Some CalDAV servers like Nextcloud, iCloud and Google Calendar and probably others already support that feature ([Webcal subscriptions](https://docs.nextcloud.com/server/latest/admin_manual/groupware/calendar.html#subscriptions)). When adding such a subscription on the server, they appear in DAVx⁵ in the _Webcal_ tab (or as a normal read-only CalDAV calendar, when the server chooses this approach). Maybe this could be an option for you.

**Reply by sihma** [reply]
Yes. Sadly subscriptions are often not imported into the existing calendar, but as a side-to-side read-only calendar. Manual import often imports in the existing calendar, but there is no automatic import.

But thinking more through it, a feature like this on ICSx5 would clutter the software, be a lot of work and would only suit my use case. Sorry for asking :)
It's better added to Cal.com software where it'd be a super useful functionality! I'll see what their answer is.

Thanks for your time!
